tests/integration-tests/tests/update/test_update.py

```diff
@@ -490,6 +495,40 @@ def _check_role_attached_policy(region, cluster, policy_arn):
     assert policy_arn in policies
 
 
+def get_ebs_volume_ids(cluster, region):
+    # get the list of configured ebs volume ids
+    # example output: ['vol-000', 'vol-001', 'vol-002']
+    ebs_stack = utils.get_substacks(cluster.cfn_name, region=region, sub_stack_name="EBSCfnStack")[0]
+    return utils.retrieve_cfn_outputs(ebs_stack, region).get("Volumeids").split(",")
+
+
+def describe_volume(volume_id, region):
+    volume = boto3.client("ec2", region_name=region).describe_volumes(VolumeIds=[volume_id]).get("Volumes")[0]
+    volume_type = volume.get("VolumeType")
+    volume_iops = volume.get("Iops")
+    volume_throughput = volume.get("Throughput")
+    return volume_type, volume_iops, volume_throughput
+
+
+def _check_volume(cluster, config, region):
+    logging.info("checking volume throughout and iops change")
+    volume_ids = get_ebs_volume_ids(cluster, region)
+    volume_type = config.get("ebs custom", "volume_type")
+    volume = describe_volume(volume_ids[0], region)
```

Comment:
For clarity, we can also do `volume_type, volume_iops, volume_throughput = describe_volume(volume_ids[0], region)`.

This way we know immediately which attribute we are asserting later, instead of having to figure out which attribute is `volume[1]`

Reply:
Done